examples/pytorch/setup.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -Eeo pipefail
+
+pre_run () {
+  pip install torchvision
```

Comment:
Do we need this `torchvision` install just for the dataset?

Reply:
Yes, its the easiest way. Any suggestions to the contrary?

Reply:
If we do a `pip install`, this means that `integration uninstall` will not properly clean up and the user will be left with a `torchvision` installed.

Perhaps we have [one of these](https://github.com/zalandoresearch/fashion-mnist#loading-data-with-other-machine-learning-libraries) libraries installed. Maybe grab it via Tensorflow?

Reply:
No better suggestion unfortunately other than getting it from somewhere else (or using a dataset that somehow comes with base `torch` if there are any). @strickvl Requiring tensorflow in our pytorch example might not be optimal either I think